api/manifest/manifest.go

```diff
@@ -1448,6 +1448,20 @@ func initApplication(dir string) error {
 		return err
 	}
 
+	if kind == "rails" {
```

Comment:
See comments above about files from images becoming unavailable with volume mounts.

This should totally be a switch. I'll make that change and refactor this a bit, but I think we still need to generate these files if volume mounting is our sync solution going forward.